src/components/Sum.js

```diff
@@ -0,0 +1,17 @@
+import { number } from 'prop-types';
+import React from 'react';
+
+Sum.propTypes = {
+  a: number.isRequired,
+  b: number.isRequired,
+};
+
+function Sum({ a = 0, b = 0 }) {
```

Comment:
You either make `a` and `b` required, or give them default values using `defaultProps`. Second option is more stable.